frontend/awx/administration/applications/ApplicationForm.tsx

```diff
@@ -47,10 +47,10 @@ export function CreateApplication(props: { onSuccessfulCreate: (app: Application
   return (
     <PageLayout>
       <PageHeader
-        title={t('Create Application')}
+        title={t('Create OAuth application')}
         breadcrumbs={[
-          { label: t('Applications'), to: getPageUrl(AwxRoute.Applications) },
-          { label: t('Create Application') },
+          { label: t('OAuth Applications'), to: getPageUrl(AwxRoute.Applications) },
+          { label: t('Create OAuth application') },
```

Comment:
There is OAuth application caption, while in buttons, there are Create application or Edit application without OAuth name, is it intentional?

Reply:
I think this and the above should be fixed in this PR I'm working on: https://github.com/ansible/ansible-ui/pull/2753